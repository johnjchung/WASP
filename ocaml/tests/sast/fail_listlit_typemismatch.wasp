Endpoint temp(GET:) {
    list<int> a = ["hello", 1];
}
