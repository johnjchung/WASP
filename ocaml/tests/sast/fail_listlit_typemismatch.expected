Fatal error: exception Type_checking.ListLitTypeError("A LIST LITERAL CANNOT BE DECLARED WITH EXPRESSIONS OF MIXED TYPES: string AND int")
